src/Provider/AbstractProvider.php

```diff
@@ -581,12 +612,12 @@ protected function parseResponse(ResponseInterface $response)
      * Custom mapping of expirations, etc should be done here. Always call the
      * parent method when overloading this method!
      *
-     * @param  array $result
+     * @param  mixed $result
```

Comment:
`parseResponse` can return a string, *but* if a uid key is to be set and the parse didn't result in an array, that means the parse is broken. Not sure how to indicate that.